packages/camera/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.5.0
+
+* **Breaking Change** This plugin no longer handles closing and opening the camera on Android
+  lifecycle changes. Please use `WidgetsBindingObserver` to control camera resources on lifecycle
```

Comment:
Could you include some sample code here showing how people are expected to control the camera going forward?

Reply:
As discussed offline, example was added to example project.